service_layer_test.cc

```diff
@@ -221,6 +221,59 @@ TEST(ServiceLayerFunctionalityTest, ValidParentID) {
   EXPECT_EQ(sl_func.valid_parent_id("5"), false);
 }
 
+TEST(ServiceLayerFunctionalityTest, Stream) {
+  ServiceLayerFunctionality sl_func(true);
+  sl_func.registeruser("jillian");
+  EXPECT_EQ(sl_func.user_exists("jillian"), true);
+  sl_func.startStream("jillian", "#Athenahacks");
+
+  Chirps chirps1 = sl_func.stream("jillian");
+  // stream should not initially return a chirp
+  EXPECT_EQ(chirps1.chirps_size(), 0);
+
+  Chirp *chirp1 = new Chirp();
```

Comment:
Minor: you might use unique_ptr here and below.

Reply:
I forgot to fork the repository instead of just being on Krishna's repo but done on the forked version (new pr to come I am so sorry)